src/test/java/org/broadinstitute/consent/http/resources/DatasetResourceTest.java

```diff
@@ -808,6 +808,15 @@ void testFindAllDatasetsAvailableToUserAsCustodian() {
     assertEquals(HttpStatusCodes.STATUS_CODE_OK, response.getStatus());
   }
 
+  @Test
+  void testFindAllDatasetStudySummaries() {
+    when(userService.findUserByEmail(any())).thenReturn(user);
+    when(datasetService.findAllDatasetStudySummaries()).thenReturn(List.of());
+    initResource();
+    Response response = resource.findAllDatasetStudySummaries(authUser);
+    assertEquals(HttpStatusCodes.STATUS_CODE_OK, response.getStatus());
```

Comment:
Is there any value in asserting that the expected dataset summaries are returned, perhaps worth exercising in a different test case than the empty summary list one?

Reply:
Personally, I don't find value in adding that unless the called method, `findAllDatasetStudySummaries`, performed some kind of logical operation on the what the service layer returned. In this case, we're returning whatever the service layer provides regardless of what it is. Similarly, in this case, the service layer is also doing a plain pass-through to the DAO layer. That's where we really care about what the entities look like.